src/engine/enginebuffer.cpp

```diff
@@ -1136,7 +1151,7 @@ void EngineBuffer::processSlip(int iBufferSize) {
     bool enabled = static_cast<bool>(load_atomic(m_slipEnabled));
     if (enabled != m_bSlipEnabledProcessing) {
         m_bSlipEnabledProcessing = enabled;
-        if (enabled) {
+        if (enabled || m_slipCancelled.fetchAndStoreAcquire(0)) {
```

Comment:
This will not work, because we need to detect the push event. Otherwise, it might happen that a short push release
cycle is slipping trough.